jupyterlab_git/git.py

```diff
@@ -110,12 +112,20 @@ def call_subprocess(
         cmdline: "List[str]",
         cwd: "Optional[str]" = None,
         env: "Optional[Dict[str, str]]" = None,
+        is_binary=is_binary,
     ) -> "Tuple[int, str, str]":
         process = subprocess.Popen(
             cmdline, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=cwd, env=env
         )
         output, error = process.communicate()
-        return (process.returncode, output.decode("utf-8"), error.decode("utf-8"))
+        if is_binary == False:
+            return (process.returncode, output.decode("utf-8"), error.decode("utf-8"))
+        else:
+            return (
+                process.returncode,
+                base64.encodebytes(output).decode("ascii"),
+                error.decode("utf-8"),
+            )
```

Comment:
let's make it simpler :wink: 

```suggestion
        if is_binary:
            return (
                process.returncode,
                base64.encodebytes(output).decode("ascii"),
                error.decode("utf-8"),
            )
        else:
            return (process.returncode, output.decode("utf-8"), error.decode("utf-8"))
```